generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,10 @@
+# ===================================================================
+# Spring Cloud Config bootstrap configuration
+# ===================================================================
+
+spring:
+    application:
+        name: <%= baseName %>
+    cloud:
+        config:
+            uri: http://localhost:8761/config
```

Comment:
@gmarziou I think your first proposal is much simpler. I can easily add a standalone profile that will disable the eureka and config client.
 I'm sure we could embed a eureka server and a config server only for a specific profile but for integration tests, shouldn't starting the registry with docker be good enough ?